example/package.json

```diff
@@ -3,7 +3,8 @@
   "version": "0.0.1",
   "private": true,
   "scripts": {
-    "build:ios": "mkdirp dist && react-native bundle --entry-file index.js --platform ios --dev true --bundle-output dist/main.jsbundle --assets-dest dist --reset-cache"
+    "build:android": "mkdirp dist && mkdirp dist/res && react-native bundle --entry-file index.js --platform android --dev true --bundle-output dist/main.jsbundle --assets-dest dist/res --reset-cache",
```

Comment:
Why do the assets need to be inside `dist/res`?

Reply:
The assets option is ambiguous. In fact, it does not correspond to android assets, but rather to resources that should go under `res` folder.